app/views/ChargeSummary.scala.html

```diff
@@ -278,11 +282,11 @@ <h1 class="govuk-heading-xl">
         @paymentProcessingInfo
     }
 
-    @if(isAgent && documentDetail.remainingToPay > 0 && !(codingOutEnabled && documentDetail.isPayeSelfAssessment)){
+    @if(isAgent && documentDetail.remainingToPay > 0 && !(codingOutEnabled && documentDetail.isPayeSelfAssessment) && !isBalancingChargeZero){
```

Comment:
we don't need to show payment options in the case of a zero amount balancing charge (nothing to pay) - so existing logic should work without changes, no?

Reply:
right. will make the change